gravitl/netmaker
# update logic for determing when to check for free port

Test Steps:

1. create two non-udp hole punch networks
2. have node join both networks (note listening port for both interfaces)
3. change the listening port of the node on network2 to the listening port of network1
** verify the listening port of the node on network 2 remains the same
4. do netclient pull on network2
** verify the listening port of the node on network 2 remains the same
